context.satyh

```diff
@@ -0,0 +1,76 @@
+@import: inline
+
+type font = string * float * float
```

Comment:
It's not documented in code what each member in the `font` type stands for.
We may want to do either of
* make `font` type as a record type (something like `(|font-name: string; ratio: float; baseline: float|)` )
* Provide `Font` module with `Font.t` and `Font.make(font-name, ratio, baseline)`

Reply:
If you think it's better to align with the original API for `ctx`, I'm also okay with this code.

Reply:
The font type is something the satysfi runtime internally uses and not what I newly introduced (the name “font” is indeed used to refer that type alias).
Hence I do not wish to take the fist option of yours as this pr is mainly a set of reexports and not more.
I will add some comments ahead of the file. Thx 😊